src/bundler.ts

```diff
@@ -79,7 +79,7 @@ class Bundler {
                     String(opts.abspath).trim() !== '') ?
         path.resolve(opts.abspath) :
         undefined;
-    this.pathResolver = new PathResolver(Boolean(this.abspath));
+    this.pathResolver = new PathResolver(this.abspath);
```

Comment:
+1 to PathRewriter.

I'd even be up for a more verbose name like UrlAttributeRewriter.